### PR TITLE
Add documentation for adding new layers to Fabrik

### PR DESCRIPTION
**Pull request by haabhi**
This PR adds documentation for adding new layers to Fabrik for keras and tensorflow.

### CODE REVIEW

File: tutorials/adding_new_layers.md [hunk 1]
```diff
@@ -54,33 +57,200 @@
...
+
+#### Importing a layer
+
+- Open [keras_app/views/layers_import.py](https://github.com/Cloud-CV/Fabrik/blob/master/keras_app/views/layers_import.py).
+    - Add a function to process the new layer.
+        - The function should take a Keras `Layer` object, called `layer`.
+        - You can get the layer parameters from `layer`, and create a json layer with the function  [jsonLayer](https://github.com/Cloud-CV/Fabrik/blob/master/keras_app/views/layers_import.py#L529). 
+        - Return the json layer you just created.
+
+        ```
```
Comment (Ram81) [inline tutorials/adding_new_layers.md]: Please maintain consistency in style for whole document. As there are images in above explanation for code can you either add images here or change the other images to code.

Comment (haabhi) [reply tutorials/adding_new_layers.md]: I've changed all the images to texts whenever possible

**Comment by haabhi** [comment]
@Ram81 Made the changes, please review :)

File: tutorials/adding_new_layers.md [hunk 2]
```diff
@@ -20,71 +31,140 @@
...
 
-- Add the export function in the ```layer_map``` array.
 
-### Testing and pushing the new layer.
+## Testing and pushing the new layer.
 
 - Run the fabrik application on you local machine by following the instructions in [README](https://github.com/Cloud-CV/Fabrik/blob/master/README.md) file.
 
-<img src="https://raw.githubusercontent.com/Cloud-CV/Fabrik/master/tutorials/layertesting.png" />
+<img src="https://raw.githubusercontent.com/Cloud-CV/Fabrik/master/tutorials/layer_testing_dropout.png" />
```
Comment (Ram81) [inline tutorials/adding_new_layers.md]: This image is not visible in markdown, can you fix this

Comment (haabhi) [reply tutorials/adding_new_layers.md]: @Ram81 The image should be visible once this branch is merged. This is the image: https://raw.githubusercontent.com/haabhi/Fabrik/doc_add_layer/tutorials/layer_testing_dropout.png

**Pull request merged**